tests/test.js

```diff
@@ -121,4 +123,23 @@ describe('Verifies async features', function() {
     expect(console.log.getCall(13).args[0]).to.equal('Alias -> Command');
     expect(console.log.getCall(14).args[0]).to.equal('================\n');
   });
+
+  it('prints version', function() {
+    squash(VERSION_FLAG);
+    assert.isTrue(console.log.called, 'log should have been called.');
+    expect(console.log.getCall(0).args[0]).to.equal('\n');
+    expect(console.log.getCall(1).args[0]).to.equal('#########################################################################################');
+    expect(console.log.getCall(2).args[0]).to.equal('####       ######        #####  ########  #####        ########        ####  ########  ##');
+    expect(console.log.getCall(3).args[0]).to.equal('###  ###########  ######  ####  ########  ####  ######  #####  ############  ########  ##');
+    expect(console.log.getCall(4).args[0]).to.equal('##  ###########  ########  ###  ########  ###  ########  ###  #############  ########  ##');
+    expect(console.log.getCall(5).args[0]).to.equal('###  ##########  ########  ###  ########  ###  ########  ####  ############  ########  ##');
+    expect(console.log.getCall(6).args[0]).to.equal('####       ####  ########  ###  ########  ###  ########  #####         ####            ##');
+    expect(console.log.getCall(7).args[0]).to.equal('#########   ###  ########  ###  ########  ###            ############  ####  ########  ##');
+    expect(console.log.getCall(8).args[0]).to.equal('##########  ####  ######  ####  ########  ###  ########  #############  ###  ########  ##');
+    expect(console.log.getCall(9).args[0]).to.equal('#########  ######       #######  ######  ####  ########  ############  ####  ########  ##');
+    expect(console.log.getCall(10).args[0]).to.equal('###       #############  ########      ######  ########  ####        ######  ########  ##');
+    expect(console.log.getCall(11).args[0]).to.equal('#########################################################################################');
+    expect(console.log.getCall(12).args[0]).to.equal('\n');
+    expect(console.log.getCall(13).args[0]).to.contain('Squash version is '); // We don't want to check for the actual version number to avoid changing this every release
```

Comment:
This needs to be modified based on above changes.
For dynamic versions check, you can import package.json in the test file and then can match the version with the printed version